### DISCUSSION TITLE
Specify number of lags?

### DISCUSSION THREAD

**Discussion by michal-sankot** [google/lightweight_mmm#250] [Q&A]
In our domain, carryover lag is expected to be around 14 days. Where could I specify that number of lags should be 14 and not the default 13*7 days?

Thanks

**Comment by michal-sankot** [comment]
I hacked it like this. But it doesn't look like the most elegant solution as it overrides internal attributes. Is there some proper way?

```
def my_function(media_data, custom_priors):
  return models.transform_carryover(media_data=media_data, custom_priors=custom_priors, number_lags=14)

mmm = lightweight_mmm.LightweightMMM(model_name="carryover")
mmm._model_transform_function = my_function
```

**Reply by akim-collab** [reply]
Thank you very much! It really seems like a fairly quick solution to this problem)
